File: connectors/php/examples/profile_test.php
<?php

declare(strict_types=1);

require __DIR__ . '/profile.php';

$actual = renderProfile(['id' => 'guest']);
$expected = 'Profile: GUEST';

if ($actual !== $expected) {
    fwrite(STDERR, "expected {$expected}, got {$actual}\n");
    exit(1);
}

echo "php profile tests passed\n";

File: connectors/go/examples/http/profile_test.go
package main

import "testing"

func TestRenderProfileFallback(t *testing.T) {
	got := renderProfile(profileUser{ID: "guest"})
	want := "Profile: GUEST"

	if got != want {
		t.Fatalf("renderProfile() = %q, want %q", got, want)
	}
}
